Sources/AnimatedTabBar/TabBarButtons/WiggleButton.swift

```diff
@@ -9,14 +9,16 @@ import SwiftUI
 
 public struct WiggleButton: View {
 
-    public var image: Image
-    public var maskImage: Image
-    public var isSelected: Bool
-
-    public init(image: Image, maskImage: Image, isSelected: Bool) {
-        self.image = image
-        self.maskImage = maskImage
-        self.isSelected = isSelected
+    var image: Image
+    var maskImage: Image
+    var imageSize: CGFloat
+    var isSelected: Bool
+    
+    init(image: Image, maskImage: Image, imageSize: CGFloat? = nil, isSelected: Bool) {
+      self.image = image
```

Comment:
Please preserve the chosen inset style

Reply:
done i think, if you didn't mean the order.